doc/content/getting-started/migrating-from-networks/migrate-from-chirpstack.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Migrate from ChirpStack"
+description: ""
+weight: 3
+---
+
+This section contains instructions on how to migrate end devices from ChirpStack to {{% tts %}}.
+
+<!--more-->
+
+End devices and applications can easily be migrated from ChirpStack to {{% tts %}} with the `ttn-lw-migrate` tool. This tool is used for exporting end devices and applications to a [JSON file]({{< ref "getting-started/migrating-from-networks/device-json.md" >}}) containing their description. This file can later be imported in {{% tts %}} as described in the [Import End Devices in The Things Stack]({{< ref "getting-started/migrating-from-networks/import-devices.md" >}}) section.
+
+First, configure the environment with the following variables modified according to your setup:
+
+```bash
+$ export CHIRPSTACK_API_URL="localhost:8080"    # ChirpStack Application Server URL
+$ export CHIRPSTACK_API_TOKEN="7F0as987e61..."  # ChirpStack API key
+$ export JOIN_EUI="0101010102020203"            # Set JoinEUI for exported devices
+$ export FREQUENCY_PLAN_ID="EU_863_870"         # Set FrequencyPlanID for exported devices
+```
```

Comment:
Possibly worth noting that `export CHIRPSTACK_API_INSECURE=1` is needed if the ChirpStack is served without TLS.